src/main/webapp/jsp/instructorComments.jsp

```diff
@@ -29,7 +27,7 @@
                 </h4>
                 <comments:commentsNotifyPanel courseId="${data.courseId}" numberOfPendingComments="${data.numberOfPendingComments}" />
             </div>
-            <div id="no-comment-panel" style="${empty data.comments && empty data.feedbackSessions ? '' : 'display:none;'}">
+            <div id="no-comment-panel" style="${empty data.commentsForStudentsTables && empty data.feedbackSessions ? '' : 'display:none;'}">
```

Comment:
You can use EL here for 'and' instead of && to be consistent with 'not', 'empty', etc.